RoboJackets/robocup-software
# Neutralize default defense

## Description
The standard defense that we use is incompatible with our current systems on a number of levels. This PR removes the clearing functionality from defense.py and renames it coordinated_block.py to better reflect what it does, and removes the standard defense code from standard_play.py

There are some places that probably should be using coordinated block but aren't, but it's a simple matter of adding the sub behavior and setting the handles goalie function to true.

## Associated Issue
The critical issue #1338 should be pretty much fixed in it's current incarnation although not entirely.

## Steps to test
### Test Case 1
1. Run the altered plays
2. See if it works

Expected result: Things generally work and coordinated_block is only activated when called
